src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-CSharp/Controllers/ValuesController.cs

```diff
@@ -18,14 +18,14 @@ public class ValuesController : ControllerBase
     {
         // GET api/values
         [HttpGet]
-        public IEnumerable<string> Get()
+        public ActionResult<IEnumerable<string>> Get()
```

Comment:
This one is kinda meh.

Reply:
@davidfowl care to elaborate?

Reply:
Nested generics don't look great. It's also no async (I assume on purpose).

Reply:
Well of course this method can't  be async, it's trivially sync...

And the developer is expected to modify this method to their suiting, so they can change the return type and async-ness as needed once they fill in their code.

Reply:
```C#
public Task<ActionResult<IEnumerable<string>>> Get() => Task.FromResult(new [] { "zomg" });
```

Reply:
Not going to feed the TROLLS. You troll.